docs_src/training.ipynb

```diff
@@ -27,29 +27,29 @@
     "hide_input": false
    },
    "source": [
-    "The fastai library is structured training around a [`Learner`](/basic_train.html#Learner) object that binds together a pytorch model, some data with an optimizer and a loss function, which then will allow us to launch training.\n",
+    "The fastai library structures its training process around the [`Learner`](/basic_train.html#Learner) class, whose object binds together a PyTorch model, a dataset, an optimizer, and a loss function; the entire learner object then will allow us to launch training.\n",
     "\n",
-    "[`basic_train`](/basic_train.html#basic_train) contains the definition of this [`Learner`](/basic_train.html#Learner) class along with the wrapper around pytorch optimizer that the library uses. It defines the basic training loop that is used each time you call the [`fit`](/basic_train.html#fit) function in fastai (or one of its variants). This training loop is kept to the minimum number of instructions, and most of its customization happens in [`Callback`](/callback.html#Callback) objects.\n",
+    "[`basic_train`](/basic_train.html#basic_train) defines this [`Learner`](/basic_train.html#Learner) class, along with the wrapper around the PyTorch optimizer that the library uses. It documents the basic training loop that is used each time you call the [`fit`](/basic_train.html#fit) function(or one of its variants) in fastai. A training loop method requires an intentionally minimal number of arguments; if you want to customize your training loop, you could include in the arguments an optional [`Callback`](/callback.html#Callback) object, which provides a rich number of customization options for the training loop.\n",
     "\n",
-    "[`callback`](/callback.html#callback) contains the definition of those, as well as the [`CallbackHandler`](/callback.html#CallbackHandler) that is responsible for the communication between the training loop and the [`Callback`](/callback.html#Callback) functions. It maintains a state dictionary to be able to provide to each [`Callback`](/callback.html#Callback) all the informations of the training loop, easily allowing any tweaks you could think of.\n",
+    "[`callback`](/callback.html#callback) defines these customization options and the [`CallbackHandler`](/callback.html#CallbackHandler) that is responsible for the communication between the training loop and the [`Callback`](/callback.html#Callback)'s methods. The [`CallbackHandler`](/callback.html#CallbackHandler) maintains a state dictionary able to provide each [`Callback`](/callback.html#Callback) object all the information of the training loop it belongs to, making any imaginable tweaks of the training loop within your reach.\n",
```

Comment:
Please remove "[`callback`](/callback.html#callback) defines these customization options" as it's not really true.  Just say it defines this class.
At the end, I'd put "of this training loop" (not the)

Reply:
Thanks!